### PR TITLE
Moving more action related files to actions modules

### PR DESCRIPTION
**Pull request by hkaiser**
- flyby: remove unused manage_object_action
- flyby: remove obsolete header reference from build system

This doesn't necessarily has to go into V1.5

### CODE REVIEW

**Comment by msimberg** [comment]
> @msimberg, @aurianer unexpectedly this looks good much faster than anticipated. I think this can still go into the release.

Yep. CSCS's jenkins is running again, but playing a bit of catchup with all the builds that got queued up. Let's see if it manages to get to this one soon before merging. I'm also looking into the rostam mpi failures.

**Comment by hkaiser** [comment]
@msimberg, @aurianer could we still merge this? Everything looks quite ok now.

**Comment by msimberg** [comment]
> @msimberg, @aurianer could we still merge this? Everything looks quite ok now.

This still needs some attention: https://cdash.cscs.ch/viewBuildError.php?buildid=124497. That's with the distributed runtime off, so you may just need to make some module dependencies conditional.

File: libs/include/CMakeLists.txt [hunk 1]
```diff
@@ -122,6 +122,7 @@ add_hpx_module(
   GLOBAL_HEADER_GEN OFF
   HEADERS ${include_headers}
   DEPENDENCIES
+    hpx_actions
```
Comment (msimberg) [inline libs/include/CMakeLists.txt:125]: I think this is the most likely culprit for the build failure.

**Review by msimberg — APPROVED** [review]
@hkaiser looks good to me btw! Sorry I wasn't more explicit earlier. If CI is happy with this let's absolutely get this in the release.

**Review by aurianer — APPROVED** [review]

**Comment by hkaiser** [comment]
> > @msimberg, @aurianer could we still merge this? Everything looks quite ok now.
> 
> This still needs some attention: https://cdash.cscs.ch/viewBuildError.php?buildid=124497. That's with the distributed runtime off, so you may just need to make some module dependencies conditional.

That's fixed now. Thanks for noticing.

**Comment by msimberg** [comment]
@hkaiser thanks for updating this. Looks good now!

**Pull request merged**